airflow/config_templates/config.yml

```diff
@@ -1178,6 +1197,7 @@ api:
       version_added: 2.2.0
       example: ~
       default: ""
+
```

Comment:
Seems they were added by some auto-formatting. Can we remove those empty lines back?

Reply:
Actually, I added them manually 🙃 . I observed in some places we had empty new lines between these config sections, in some places no empty new lines to separate the sections and in 1-2 occurrences multiple empty new lines between sections. How would we like to keep it?
Remove the empty new line separators between all sections?

Reply:
Removed empty new lines from everywhere else to keep it consistent.